.github/workflows/wiki-mirror.yml

```diff
@@ -8,8 +8,12 @@ on:
     branches:
       - master
 
+permissions: {}
```

Comment:
Out of curiosity, is `permissions: {}` needed here? If yes, shouldn't be at `ci.yml` as well?